code/datums/uplink_item.dm

```diff
@@ -972,6 +973,14 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	cost = 5
 	gamemodes = list(/datum/game_mode/nuclear)
 
+/datum/uplink_item/stealthy_weapons/combat_minus
+	name = "Experimental Krav Gloves"
+	desc = "Experimental gloves with installed nanochips that teach you Krav Maga when worn, great as a cheap backup weapon. Warning, the nanochips will override any other fighting styles such as CQC. Do not look as fly as the Warden's"
```

Comment:
I would make this clear to the purchaser that these aren't visible on examine. other than that. looks good